servers/rendering/renderer_rd/renderer_scene_render_rd.cpp

```diff
@@ -2362,6 +2362,8 @@ void RendererSceneRenderRD::_render_buffers_post_process_and_tonemap(const Rende
 
 	// @TODO IMPLEMENT MULTIVIEW, all effects need to support stereo buffers or effects are only applied to the left eye
 
+	float resolution_factor = p_render_data->cam_vaspect ? (float(rb->internal_width) / 1920.0) : (float(rb->internal_height) / 1080.0);
```

Comment:
Does 1920x1080 correspond to something specific in the renderer? It seems a bit weird to see this hardcoded, wouldn't it need to reflect the actual game viewport size?

Reply:
Ah I see this was discussed in https://github.com/godotengine/godot/issues/56911#issuecomment-1019330478. A comment to justify the hardcoded reference size would be useful.

Reply:
I agree, we probably shouldn't hardcode such values. @Calinou touched upon this point in [his comment](https://github.com/godotengine/godot/issues/56911#issuecomment-1019330478).